src/ol/transform.js

```diff
@@ -45,6 +45,26 @@ export function reset(transform) {
   return set(transform, 1, 0, 0, 1, 0, 0);
 }
 
+/**
+ * @param {!Transform} transform1 Transform 1.
+ * @param {!Transform} transform2 Transform 2.
+ * @return {boolean} Transform 1 and 2 are pixel equal.
+ */
+export function equals(transform1, transform2) {
+  if (transform1 === transform2) {
+    return true;
+  }
+  const tolerance = 1e-6;
+  return (
```

Comment:
This is efficient but ideally it should compare the differences relative to the values instead of using a fixed tolerance for each element.  Since browsers will round when applying transforms (e.g. Firefox uses Float32) should we not attempt something similar, e.g. `equalsArray(transform1.map(Math.fround), transform2.map(Math.fround))`?

Reply:
Tried with `Math.fround`, but that breaks existing tests.